hooks/post/99-version_locking.rb

```diff
@@ -1,4 +1,4 @@
-if get_custom_config(:lock_package_versions)
+if package_lock_feature? && app_value(:lock_package_versions)
```

Comment:
I don't think this is needed since `app_value(:lock_package_versions)` should never be true if the feature isn't present (see the suggestion in the boot hook)